src/Compute/Compute/help/Set-AzVmssStorageProfile.md

```diff
@@ -151,7 +151,7 @@ Specifies the type of virtual machine image (VMImage) offer.
 To obtain an image offer, use the Get-AzVMImageOffer cmdlet.
 
 ```yaml
-Type: System.String
+Type: String
```

Comment:
Be sure to follow [Updating all markdown files in a module](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module). These types should have full names.

Reply:
did these changes come from use the script Priyam?

Reply:
Yes they were added as part of running the script

Reply:
What should I do, should I revert them? I did not make these changes manually.